web/themes/new_weather_theme/templates/partials/wind.html.twig

```diff
@@ -6,13 +6,14 @@
   #}
   <div class="display-flex flex-row flex-align-center" aria-hidden="true">
     <span>{{wind.speed}} {{ "mph" | t }}</span>
+    <span class="padding-x-05">{{ wind.direction.short }}</span>
     {#
     The +180 here is to account for the fact that wind direction
     is reported as a FROM direction, but our arrow points in the
     TO direction. So we just need to spin it 'round.
     #}
-    <svg role="img" aria-hidden="true" data-wind-direction class="width-2 height-2 margin-left-05 padding-2px" style="transform: rotate({{ wind.direction.angle + 180 }}deg);">
-      <use xlink:href="{{ "/" ~ directory ~ "/assets/images/spritesheet.svg#wind_arrow_s" }}"></use>
+    <svg role="img" aria-hidden="true" data-wind-direction class="width-2 height-2 padding-2px" style="transform: rotate({{ wind.direction.angle + 180 }}deg);">
```

Comment:
It looks like putting padding on the actual svg element makes the vector smaller because the padding counts as part of the size, and makes it a little harder to align. So I suggest we warp the svg in a span that we can then add padding and alignment classes to:

```suggestion
      <span class="display-flex flex-align-center padding-bottom-2px">
    <svg role="img" aria-hidden="true" data-wind-direction class="width-2 height-2" style="transform: rotate({{ wind.direction.angle + 180 }}deg);">
```